cygal/src/lib/PhotoPyrenees.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  const { position, title, date, body, src, alt, ...restProps } = $$props;
+</script>
+
+<div class="flex gap-4s max-w-1xl pb-16">
+  <div
+    class={position === 'right'
```

Comment:
it's great to have this functionality! Maybe that comparing to a string is however prone to bugs, imagine if the input json is malformed and has `rigt`. Then there would be an unexpected behavior for the user.

Maybe declaring the enums that you had and using those is more robust. The json parser could assign the right enums and ensure that the json is properly validated.

Reply:
maybe adds this to the ticket on json loading for the future